src/component/material-ui/nav-wrapper/index.js

```diff
@@ -173,5 +214,5 @@ const mapDispatchToProps = dispatch => ({
   logout: () => dispatch(logoutAction()),
 });
 
-export default connect(mapStateToProps, mapDispatchToProps)(NavWrapper);
+export default withRouter(connect(mapStateToProps, mapDispatchToProps)(NavWrapper));
```

Comment:
withRouter gives it access to `this.props.location`?

Reply:
yes, it does this for the nav-wrapper so it has access since it is not in a Route